app/src/main/java/com/example/peterchu/watplanner/home/HomePresenter.java

```diff
@@ -68,4 +80,62 @@ public boolean onSubmitSearchQuery(String subject, String catalogNumber) {
         Course c = dataRepository.getCourse(subject, catalogNumber);
         return homeFragment.openDetailView(c == null ? null : c.getId());
     }
+
+    public void onExportCoursesToCalendar(List<CourseComponent> courseSchedule) {
+        for (CourseComponent courseComponent : courseSchedule) {
+            long calID = getDefaultCalendarID(this.homeFragment.getActivity());
+            Calendar beginTime = Calendar.getInstance();
+            beginTime.set(2016, 0, 19, 7, 30);
+            long startMillis = beginTime.getTimeInMillis();
+            Calendar endTime = Calendar.getInstance();
+            endTime.set(2016, 0, 19, 8, 30);
+            long endMillis = endTime.getTimeInMillis();
+
+            ContentResolver cr = this.homeFragment.getActivity().getContentResolver();
+            ContentValues values = new ContentValues();
+            values.put(CalendarContract.Events.DTSTART, startMillis);
+            values.put(CalendarContract.Events.DTEND, endMillis);
```

Comment:
oh yeah, i should parse that from the course component, I was testing it with this 2016 date and it works. I'll update this PR in a bit